dbld/builddeps

```diff
@@ -157,14 +157,17 @@ function install_pip_packages {
         fedora-*)
```

Comment:
checked: fedora-33 and fedora-34 both have "python3". So we do not need the "python" alias anymore. (We can leave it to the default branch.)